model-api/src/main/java/edu/unc/lib/boxc/model/api/sparql/SparqlUpdateHelper.java

```diff
@@ -183,7 +183,11 @@ public static String getObjectAsString(Object object) {
         } else if (object instanceof Literal) {
             Literal literal = (Literal) object;
             RDFDatatype type = literal.getDatatype();
-            return '"' + literal.getLexicalForm() + "\"^^<" + type.getURI() + ">";
+            if (type.getURI().equals("http://www.w3.org/2001/XMLSchema#string")) {
```

Comment:
For safety in case the type is ever null, i'd suggest
```
if (type == null || type.getURI().equals("http://www.w3.org/2001/XMLSchema#string")) {
```
I feel like there's probably a constant out there somewhere for xml:string, but I don't know where one is